card/mod/06_bootstrap/set/all/rich_bootstrap.rb

```diff
@@ -1,9 +1,8 @@
 format :html do
   
   view :closed do |args|
-   closed_frame args.merge(:content=>true, :body_class=>'closed-content', :toggle_mode=>:close, :optional_toggle=>:show ) do
-      _optional_render :closed_content, args
-    end
+    args.merge! :body_class=>'closed-content'
+    super args
```

Comment:
I don't think we can make closed-content work with panel body; the before and after stuff breaks things.  It makes some sense, because closed view is a fairly substantial departure from the bootstrap panel model.